packages/scheduler/src/SchedulerMinHeap.js

```diff
@@ -0,0 +1,92 @@
+/**
```

Comment:
Does this mean we've given up on having a single easy to reason about file?

Reply:
Personally I don’t think it’s worth it but I can move this into the same file if you want me to

To maximize adoption, I think polyfilling the native API is the better approach.